Comp2023/src/main/java/frc/robot/subsystems/Arm.java

```diff
@@ -162,6 +174,9 @@ public void periodic( )
   {
     // This method will be called once per scheduler run
 
+    // TO-DO: if disabled, set LED when down
```

Comment:
This is a leftover from the climber--we don't currently use the LED when operating the arm.